cypress/support/fragments/data-export/dataExportResults.js

```diff
@@ -13,5 +13,24 @@ export default {
       }),
       MultiColumnListCell({ 'row': 0, 'column': defaultJobProfile }).exists(),
     ]);
-  }
+  },
+
+  verifySuccessExportResultCells() {
+    const dateString = /\d{1,2}\/\d{1,2}\/\d{4},\s\d{2}:\d{2}\s\w{2}/gm;
+    return cy.do([
+      MultiColumnListCell({ 'row': 0, columnIndex: 1 }).is({ content: 'Completed' }),
```

Comment:
can be useful to define constants with interactors and meaning names. It would be simpler to analyze.